client/src/components/clientlayout/forms/Form.css

```diff
@@ -0,0 +1,4 @@
+select {
+  width: 124px;
+  height: 35px;
+}
```

Comment:
you cannot have a specific tag css
As react at end merge each separate file in one and all the select tag used in the project will use this styling


you can give id or class to entire form and can use this way ahead
#form select or .form select
This way it will only trigger specific section